lib/src/main/res/layout/list_item_color.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Note for @Mauker1 : I added this TextView to display the color name, but I realized we don't have a good way to get that yet. I'm going to make another ticket for this in the future. In the mean time, it's empty and won't affect anything.